### PR TITLE
Oney without fees

### PR DESCRIPTION
**Pull request by sophie-mulard**
# ⚠️ Requirements
Reviewer, please take a look at those requirements before merging on `master` :

- [ ] Check that plugin version has been upgrated and are identical in both `composer.json` and `Helper/Config.php` files

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+            quote.shippingAddress.subscribe(function () {
+                if (self.getCode() === self.isChecked()) {
+                    self.updateOney();
+                }
+            });
+            quote.shippingAddress.subscribe(function () {
+                if (self.getCode() === self.isChecked()) {
+                    self.updateOney();
+                }
+            });
```
Comment (dan-payplug) [inline view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js:43]: Doublon ?

File: view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+        },
+        buildOneyDetail: function(oneySimulationResult){
+            var tmpDiv = $('<div/>');
+            var optionsWrapper = $('<div/>').addClass('oneyOption_wrapper');
+            var label = $.mage.__(this.getPaymentTypeLabel());
+
+            for (var i = 0; i < oneySimulationResult.options.length; i++) {
+                var optionData = oneySimulationResult.options[i];
+                var type = optionData.type;
+                var fullType = optionData.type;
```
Comment (dan-payplug) [inline view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js:187]: Doublon ?

**Review by dan-payplug — CHANGES_REQUESTED** [review]
2 minis changements, coquilles ?

File: view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js (hunk 1 above)
Comment (sophie-mulard) [reply view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js]: corrigé

File: view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js (hunk 2 above)
Comment (sophie-mulard) [reply view/frontend/web/js/view/payment/method-renderer/oney-payment-renderer.js]: corrigé

**Review by dan-payplug — APPROVED** [review]

**Pull request merged**